### DISCUSSION TITLE
failed to install glpi-agent on linux

### DISCUSSION THREAD

**Discussion by cabbo00** [glpi-project/glpi-agent#421] [Q&A]
when I run the command: sudo perl glpi-agent-1.4-linux-installer.pl it gives me the error: failed to install glpi-agent.
How can i solve it?

**Comment by g-bougard** [comment]
Hi @cabbo00 

without the full output, we can't really help you. In that case, you should even run the installation with `--verbose` option and share the output.

You also have to tell us which OS you're trying on and which version of the OS you're using.

**Comment by TheBolshe** [comment]
so i'm on a new install of ubuntu 22.04 lts, i did a sudo apt update and upgrade, this is the output when i try to install with the perl script (same with snap and no snap)

```
Installing GLPI-Agent for Linux, please wait...
Running on linux distro: Ubuntu 22.04.3 LTS : Ubuntu : 22.04.3 LTS (Jammy Jellyfish)...
glpi-agent is not installed
Cleaning...
Running: rm -rf /etc/glpi-agent /var/lib/glpi-agent
Installing glpi-agent v1.5...
Trying to install glpi-agent v1.5-1 on Ubuntu 22.04.3 LTS release (Ubuntu:22.04.3 LTS (Jammy Jellyfish))...
Extracting glpi-agent_1.5-1_all.deb ...
Extracting glpi-agent-task-collect_1.5-1_all.deb ...
Extracting glpi-agent-task-deploy_1.5-1_all.deb ...
Extracting glpi-agent-task-esx_1.5-1_all.deb ...
Extracting glpi-agent-task-network_1.5-1_all.deb ...
Running: apt -y install /home/avoeyux/Desktop/GLPI-Agent Linux/glpi-agent-task-collect_1.5-1_all.deb /home/avoeyux/Desktop/GLPI-Agent Linux/glpi-agent-task-deploy_1.5-1_all.deb /home/avoeyux/Desktop/GLPI-Agent Linux/glpi-agent-task-esx_1.5-1_all.deb /home/avoeyux/Desktop/GLPI-Agent Linux/glpi-agent-task-network_1.5-1_all.deb /home/avoeyux/Desktop/GLPI-Agent Linux/glpi-agent_1.5-1_all.deb 2>/dev/null
Reading package lists... Done
Failed to install glpi-agent
Something went wrong...abording !
```

**Reply by g-bougard** [reply]
Hi @cabbo00 

it seems you're running from a folder containing a space: `/home/avoeyux/Desktop/GLPI-Agent Linux`

Can you try from a folder without space ?

**Reply by TheBolshe** [reply]
Right, i missed that, thanks ^^

**Reply by g-bougard** [reply]
Will check anyway if the script can be improved to support folder with spaces.

**Reply by g-bougard** [reply]
Just pushed a fix in 619ad72 so next release should install on debian when perl installer is run from a folder with space.

Thank you for you report.
